### DISCUSSION TITLE
Restriction notice not visible in `cb_items` shortcode

### DISCUSSION THREAD

**Discussion by datengraben** [wielebenwir/commonsbooking#1384] [Ideas]
Anzeige aller Lastenräder über [cb_items] https://dasallrad.org/lastenraeder
![image](https://github.com/wielebenwir/commonsbooking/assets/105302830/1c82082f-03f9-44a7-8608-8989d3a2fcad)

Detailseite (Kalender) von Item "Radieschen"
![image](https://github.com/wielebenwir/commonsbooking/assets/105302830/37040057-7da9-419f-acd0-a352c9cf1149)

Detailseite (Kalender) von Item "Muskeltier" - Zeitrahmen ist deaktiviert oder nicht zugeordnet.
![image](https://github.com/wielebenwir/commonsbooking/assets/105302830/7a8d6b34-bff8-4a5e-8400-bea2e5da49a0)

**Situation:** Ich habe eine Einschränkung auf den Zeitrahmen von Radieschen gelegt und die Ausleihe über die Einschränkung unterbunden. Aktuell zeigt jetzt der `cb_items` gerenderte Teil mit einem grünen Button "Buchen" an und das Datum bis wann der Zeitrahmen eine Ausleihe theoretisch erlaubt (26.01.24). Die Auswirkung meiner Einschränkung sehe ich erst, wenn ich auf die Detail-Seite gehe. Dort ist dann im Kalender sichtbar wie lange die Einschränkung die Ausleihe über den Zeitrahmen unterbindet.

Ist das ein Fehler?

**Ich würde erwarten** das sich [cb_items] hier genauso verhält, wie als wenn ich die Einstellung über den Zeitrahmen bzw. das Item regele. Dort sehe ich ja direkt das eine Artikel-Buchung eingeschränkt ist. Also das ich bereits im von cb_items gerenderten Teil die Einschränkung der Ausleihe z.B. als Text sehe.

Ggf. könnte man das auch direkt am Shortcode konfigurieren.

**Comment by hansmorb** [comment]
Was wäre für dich erwartetes Verhalten? Dass das verfügbare Datum auf "ab 30 November 2023" gesetzt wird oder, dass eine zusätzliche Meldung in der cb_items sichtbar ist?

**Reply by datengraben** [reply]
Erwarten würde ich auf jeden Fall, dass die gerade existierende Einschränkung transparent wird, also zusätzliche Meldung in cb_items sichtbar.

Ggf. noch bis wann, also das Datum bis zum Ende der Restriction. Denn ich finde es schwierig hier generell anzunehmen für welchen Zeitraum User ein Buchung vornehmen möchten.

In Beiden Fällen könnte wegen mir der Button aber weiterhin "grün" bleiben.
